aiogram/dispatcher/handler.py

```diff
@@ -44,6 +44,10 @@ def __init__(self, dispatcher, once=True, middleware_key=None):
         self.handlers: List[Handler.HandlerObj] = []
         self.middleware_key = middleware_key
 
+    @property
+    def formatted_middleware_key(self):
+        return "pre_process_"+self.middleware_key
```

Comment:
middleware key is Optional[str]. in this case simple concatenation can make an TypeError
f-string is more safe in this case.

Also this key is used in single place it makes no sense to separately take out this attribute. Use in-place formatting instead.